### PR TITLE
Code check

### PR DESCRIPTION
**Pull request by mosidrum**
On this PR I did the following: 

- [ ] Create methods for genre(create and list)

- [ ] Add unit test for music_album class and genre class

- [ ] Correct errors in persisting data for music_album and genre class

### CODE REVIEW

File: spec/genre_spec.rb
Comment (anita00001) [inline spec/genre_spec.rb:1]: ### Hi @mosidrum 

It is great that you add test for add_item method.
I would suggest you to add other test cases for initialize, to_s, and to_json as well.

File: spec/music_album_spec.rb
Comment (anita00001) [inline spec/music_album_spec.rb:1]: ### Hi @mosidrum 

Please add test for initialize and to_json as well.

Also, describe MusicAlbum do has been repeated twice. 
It should be like this:
- describe class
   - describe method
       - it 'test case'

**Review by anita00001 — CHANGES_REQUESTED** [review]

File: classes/music_album.rb [hunk 1]
```diff
@@ -15,22 +15,15 @@ def can_be_archived?
...
-
-    File.open(file_path, 'a') do |file|
-      file.puts(music_album_data.to_json)
-    end
-  end
-
-  def save
-    SaveDataToFile.save_to_json('music_albums.json', [self])
+      'publish_date' => @publish_date,
+      'on_sportify' => @on_sportify
```
Comment (anita00001) [inline classes/music_album.rb]: I think on_sportify is a typo, shouldn't it be on_spotify ?

File: data_manager.rb [hunk 2]
```diff
@@ -60,4 +97,16 @@ def create_label_object(label)
   def create_author_object(author)
     Author.new(author['f_name'], author['l_name'])
   end
+
+  def create_game_object(game)
+    Game.new(game['publish_date'], game['multiplayer'], game['last_played_at'])
+  end
+
+  def create_music_album_object(music_album)
+    MusicAlbum.new(music_album['publish_date'], music_album['on_sportify'])
```
Comment (anita00001) [inline data_manager.rb]: I think on_sportify is a typo, shouldn't it be on_spotify?

File: modules/genre_module.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+module GenreModule
+  def create_genre
+    puts 'Enter genre name: '
+
+    print 'Name: '
+    name = gets.chomp
+
+    new_genre = Genre.new(name)
+    @genres << new_genre
+
```
Comment (anita00001) [inline modules/genre_module.rb:12]: Please associate Genre with some item.

File: modules/music_album_module.rb [hunk 4]
```diff
@@ -16,17 +16,14 @@ def create_music_album
     on_spotify = on_spotify_input == 'true'
```
Comment (anita00001) [inline modules/music_album_module.rb]: on_spotify is always set true, but that is not what expected.

Hint:
Line 15: on_spotify = gets.chomp.downcase
Remove line 16

**Review by anita00001 — CHANGES_REQUESTED** [review]

File: spec/genre_spec.rb
Comment (mosidrum) [reply spec/genre_spec.rb:1]: fix on [f9c6a1b]

File: spec/music_album_spec.rb
Comment (mosidrum) [reply spec/music_album_spec.rb:1]: fix on [f9c6a1b]

File: classes/music_album.rb (hunk 1 above)
Comment (mosidrum) [reply classes/music_album.rb]: fix on [f9c6a1b]

File: data_manager.rb (hunk 2 above)
Comment (mosidrum) [reply data_manager.rb]: fix on [f9c6a1b]

File: modules/genre_module.rb (hunk 3 above)
Comment (mosidrum) [reply modules/genre_module.rb:12]: fix on [f9c6a1b]

File: modules/music_album_module.rb (hunk 4 above)
Comment (mosidrum) [reply modules/music_album_module.rb]: fix on [f9c6a1b]

**Review by anita00001 — APPROVED** [review]
# STATUS: APPROVED ✔✔

## Hi @mosidrum 
### Your effort is evident on your code, and it is worth approving!
image

### Congratulations 🚀🎉🎉
### Keep Moving on! 🏃‍♂️

### Highlights
- Console app runs as expected ✔
- Rubocop has no offenses

#### With Best Wishes,
Anita Sharma @anita00001

**Pull request merged**